script.js

```diff
@@ -10,3 +10,191 @@ document.querySelectorAll('.mobile-link').forEach((n) => n.addEventListener('cli
+                      </ul>
+
+                    <ul class="popup-project-abilites">
+                        <li class="a1">${popData[0].technologies[0]}</li>
+                        <li class="a1">${popData[0].technologies[1]}</li>
+                        <li class="a1">${popData[0].technologies[2]}</li>
+                    </ul> 
+                    </div>
+                    
+                    <p class="mobile-popup-text">
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+                        Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+                        when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem 
+                        Ipsum is simply dummy text of the printing and typesetting industry. 
+                        Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+                        when an unknown printer took a galley of type and scrambled it 1960s with the relea
+                    </p>
+                    <div class="mobile-popup-buttons">
+                    <a href=${popData[0].linkToLiveSource}>
+                    <button class="btn">
+                        <div class="popup-btn-text">${popData[0].btn1}</div>
+                        <div class="popup-icons">
+                        <img src="portfolio-Img/see live icon.png" alt="see-live-icon">
+                        </div>
+                    </button>
+                    </a>
+                    <a href=${popData[0].linkToSource}>
+                    <button class="btn">
+                        <div class="popup-btn-text">${popData[0].btn2}</div>
+                        <div class="popup-icons">
+                        <img src="portfolio-Img/Icon -GitHub.png" alt="see-live-icon">
+                        </div>
+                    </button>
+                    </a>
+                    </div>
+                </div>`;
+
+  popWindow1.innerHTML = popScreen;
+}
+
+popDisplay();
+
+// all main cards goes there------------------------------------------------>
+
+const allSixCards = document.getElementById('all-six-cards');
+
+const cardData = [
+  {
+    id: 1,
+    title: 'Multi-Post Stories Gain+Glory',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+  },
+  {
+    id: 2,
+    title: 'Multi-Post Stories Gain+Glory',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+  },
+  {
+    id: 3,
+    title: 'Multi-Post Stories Gain+Glory',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+  },
+  {
+    id: 4,
+    title: 'Multi-Post Stories Gain+Glory',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+  },
+  {
+    id: 5,
+    title: 'Multi-Post Stories Gain+Glory',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+  },
+  {
+    id: 6,
+    title: 'Multi-Post Stories Gain+Glory',
```

Comment:
- [ ] Good Job here but as the requirement suggests, you need to store additional data into your cards array along with title and technologies, in each object : feature image, description,  link to live version and link to source.
  -  For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.